account/src/helpers/password.js

```diff
@@ -0,0 +1,11 @@
+import bcrypt from 'bcryptjs';
+
+export async function hashPassword(password) {
+    const hashedPassword = await bcrypt.hash(password, 10);
```

Comment:
I think it's a better approach to put the salt in the .env file or get it from the caller